app/components/m-header.hbs

```diff
@@ -1,87 +1,160 @@
-            <div class="auk-u-mr">
-              {{currentSession.userContent.firstName}}
-              {{currentSession.userContent.lastName}}
-              ({{currentSession.userRole}})
-            </div>
-            <a href="" {{action "logout"}}>
-              {{t "sign-out"}}
-            </a>
-          {{else}}
-            {{#link-to "login"}}
-              {{t "login"}}
-            {{/link-to}}
-          {{/if}}
         </div>
+      {{/if}}
+      <div class="auk-toolbar-complex__item">
+        <WebComponents::AuButtonLink
+                @skin="muted"
+                @icon="ki ki-circle-question">
+          {{t "help"}}
+        </WebComponents::AuButtonLink>
+        <AttachPopover
+                @renderInPlace={{true}}
+                @class="ember-attacher-popper"
+                @hideOn="clickout click"
+                @showOn="click"
+                @animation="shift"
+                @placement="bottom" {{!-- as |attacher| --}}
+        >
+          <WebComponents::AuDropdown>
+            <WebComponents::AuDropdown::Item @href="https://kaleidos.vlaanderen.be/Kaleidos_Raadplegen_20200629.pdf"
+                                             target="_blank">
+              {{t "look-at-documentation"}}
+            </WebComponents::AuDropdown::Item>
+            <WebComponents::AuDropdown::Item @href="mailto:kaleidos.servicedesk@vlaanderen.be">
+              <div>
+                <span>{{t "contact-us"}}</span>
+                <span class="auk-u-muted auk-u-text-small">{{t "email-servicedesk"}}</span>
+              </div>
+            </WebComponents::AuDropdown::Item>
+          </WebComponents::AuDropdown>
+        </AttachPopover>
+      </div>
+      <div class="auk-toolbar-complex__item auk-u-mr-3">
+        {{#if this.currentSession.userContent}}
+          <WebComponents::AuButtonLink
+                  data-test-m-header-user-actions
+                  @skin="muted"
+                  @icon="chevron-down"
+                  @layout="icon-right">
+            {{this.currentSession.userContent.firstName}}
+          </WebComponents::AuButtonLink>
+          <AttachPopover
+                  @renderInPlace={{true}}
+                  @class="ember-attacher-popper"
+                  @hideOn="clickout click"
+                  @showOn="click"
+                  @animation="shift"
+                  @placement="bottom" {{!-- as |attacher| --}}
+          >
+            <WebComponents::AuDropdown>
+              <WebComponents::AuDropdown::Item @textOnly={{true}}>
+                <WebComponents::AuAvatar @icon="user" @size="small" @name="{{this.currentSession.userContent.firstName}} {{this.currentSession.userContent.lastName}}">
+                  <p>{{this.currentSession.userContent.firstName}} {{this.currentSession.userContent.lastName}}</p>
+                  <p class="auk-u-muted auk-u-text-small">{{this.currentSession.userRole}}</p>
+                </WebComponents::AuAvatar>
```

Comment:
- klopt de size ? in prototype is het `regular`, echter lijkt daar geen css voor te bestaan. Ik vermoed dat het gebruik van regular ervoor zorgt dat we de standaard waarden gebruiken, en dan is het icoon groter.
- beetje raar dat je hier @name meegeeft, maar dat lijkt niets te doen?  (is gebruikt voor aria-label en fallback als alt parameter in <img element)
anyway, aangezien je 2 x hetzelfde doet, kan dat misschien in een getter worden gestoken.
- De avatar icon mist nog css, er is geen grijze cirkel rond, dit zou moeten komen van `auk-avatar--icon`
Maar als ik de package in jouw branch kijk zie ik dat die kleur nog op een oude css klasse staat `auk-avatart--default`
Updaten met dev en kijken of het dan wel correct is met nieuwe package (6 dagen geleden nieuwe css in deze klasse)